packages/jest-config/src/normalize.ts

```diff
@@ -800,8 +782,13 @@ export default function normalize(
           ) {
             throw createConfigError(errorMessage);
           }
+          value = oldOptions[key];
+        } else {
```

Comment:
it's guaranteed to be a string here?

(landing a runtypes or io-ts refactor for 25 would be great)

Reply:
As long as the `ValidConfig` matches the type declaration (it does here) yes

Reply:
But we trust the type declaration blindly when the object comes from IO which TS does not verify at all